rfc8446.xml

```diff
@@ -180,24 +179,14 @@ changed to separate the authentication and key exchange mechanisms from
 the record protection algorithm (including secret key length) and a hash
 to be used with the key derivation function and the Hashed Message
 Authentication Code (HMAC).</t>
```

Comment:
I can't quite place my finger on why, but the "the" in "the Hashed Message Authentication Code (HMAC)" scans weirdly to me.